docs/wearables/providers/guides/my_fitness_pal.mdx

```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
 
 ## Connecting MyFitnessPal to Vital
 
-You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-email-provider) providers.
+You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-password-provider) providers.
+
+## Before Connecting
+
+You can share your MyFitnessPal diary with Vital by locking your diary with key.
+- Go to [your diary settings](https://www.myfitnesspal.com/account/diary-settings) and click **locked with key**.
+
```

Comment:
Maybe include the mobile instructions too?

Reply:
I've added mobile instructions

Reply:
@maitham please check out the mobile instructions when you get a chance